src/driver/amdxdna/aie2_msg_priv.h

```diff
@@ -470,6 +478,37 @@ struct cmd_chain_req {
 	u32 count;
 } __packed;
 
+enum cmd_chain_npu_type {
+	CMD_CHAIN_NPU_TYPE_NON_ELF	= 0x1,
+	CMD_CHAIN_NPU_TYPE_PARTIAL_ELF	= 0x2,
+	CMD_CHAIN_NPU_TYPE_PREEMPT	= 0x3,
+	CMD_CHAIN_NPU_TYPE_MAX
+};
+
+struct cmd_chain_slot_npu {
+	union {
+		enum cmd_chain_npu_type type;
+		u32 type_u32;
+	};
```

Comment:
I don't see type_u32 was used in the code.
```suggestion
	enum cmd_chain_npu_type type;
```